server/cpho/jinja2/indicators/components/indicator_actions.jinja2

```diff
@@ -0,0 +1,26 @@
+<div class="dropdown">
```

Comment:
rather than split things out into components/, can you just put it in indicators/ and start it with an underscore, e.g. _actions.jinja2 ? It's a common pattern to start partials with an underscore